src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline

```diff
@@ -431,15 +431,16 @@ module Library
       #   require "library/v1"
       #
       #   library_service_client = Library::V1::LibraryServiceClient.new
-      #   shelf = Google::Example::Library::V1::Shelf.new
+      #   shelf = {}
```

Comment:
Hm. I think empty hash is fine so long as we update the param docs. That being said, I totally forgot to update the param docs. I'll have that done soon!